### ISSUE TITLE
How can i Use this for Blazor Wasm 

### ISSUE THREAD

**Issue by imprathamesh** [fullcalendar/fullcalendar#5865]
### Checklist

I tried to put this full calendar in blazor wasm project but its not working. 
so please tell me is compatible with blazor wasm or can make any sample project? so blazor community will understand and use it into their project.

Please mark these items with an [x]

  - [ ] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [ ] Other people will find this feature useful

### Feature Description

Please describe what this feature will do.

### Visual Mockups

If you are requesting a new UI, please include a mockup.
If not, erase this section.

**Comment by acerix** [comment]
Do you get an error message or could you explain what is not working?  It's best if you can supply a [runnable, stripped-down demonstration](http://fullcalendar.io/wiki/Reduced-Test-Cases/) showing the issue.

**Comment by contactabdulsami** [comment]
It disappears on loading after 1 second, I am facing issue with Blazor server application.

**Comment by JMackCo** [comment]
trying to Load fullcalendar from a razor page in Blazor? I'm stuck.
I can display the fullcalendar just fine and see the calendar in the index.html
But, I want to move the div id='calendar' to a  calendar.razor page inside my Blazor wasm app.

***This index.html does display the FullCalendar.
`<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />

    <title>FullCalendar Test Page</title>

    <base href="/" />
    <link href="webmanifest.json" rel="webmanifest" />
    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
    <link href="icon-512.png" rel="apple-touch-icon" sizes="512x512" />
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <link href="https://cdn.jsdelivr.net/npm/fullcalendar-scheduler@5.10.1/main.min.css" rel="stylesheet" />
    <script src="https://cdn.jsdelivr.net/npm/fullcalendar-scheduler@5.10.1/main.min.js"></script>


    

</head>

<body>

    <div id="app">Loading.......</div>
    <div id='calendar'></div>
    <div id="blazor-error-ui">
        <a href="" class="reload">Reload</a>
        <a class="dismiss">🗙</a>
    </div>
    <script src="_framework/blazor.webassembly.js"></script>
    <script>navigator.serviceWorker.register('service-worker.js');</script>

    <script>
        document.addEventListener('DOMContentLoaded', function () {
            var calendarEl = document.getElementById('calendar');
            var calendar = new FullCalendar.Calendar(calendarEl, {
                schedulerLicenseKey: "xxxx-xxx-xxxxxxxx",
                initialView: 'dayGridMonth',
                events: [
                    { id: '1', resourceId: 'b', start: '2021-12-02T02:00:00', end: '2021-12-02T02:00:00', title: 'event 1' },
                    { id: '2', resourceId: 'c', start: '2021-12-02T05:00:00', end: '2021-12-02T22:00:00', title: 'event 2' },
                    { id: '3', resourceId: 'd', start: '2021-12-06', end: '2021-12-08', title: 'event 3' },
                    { id: '4', resourceId: 'e', start: '2021-12-02T03:00:00', end: '2021-12-02T08:00:00', title: 'event 4' },
                    { id: '5', resourceId: 'f', start: '2021-12-02T00:30:00', end: '2021-12-02T02:30:00', title: 'event 5' }
                ]
            });

            calendar.render();
        });
    </script>
</body>

</html>`

I create a razor page as follows:

`@page "/"

@inject Microsoft.JSInterop.IJSRuntime jsRuntime;

    <div id='calendar'></div>

@code{
    protected async override Task OnAfterRenderAsync(bool firstRender)
    {
        if (firstRender)
        {
            await jsRuntime.InvokeAsync<object>("calendar");
            //I know the line above is not correct.... Just as close as I can get.. 
            firstRender = false;

        }
    }
}
`
And I try a local main.js... Something like This.

`window.fullcalendar = () => {
    $(function () {

        document.addEventListener('DOMContentLoaded', function () {
            var calendarEl = document.getElementById('calendar');
            var calendar = new FullCalendar.Calendar(calendarEl, {
                schedulerLicenseKey: "xxxx-xxx-xxxxxxxx",
                initialView: 'dayGridMonth',
                events: [
                    { id: '1', resourceId: 'b', start: '2021-12-02T02:00:00', end: '2021-12-02T02:00:00', title: 'event 1' },
                    { id: '2', resourceId: 'c', start: '2021-12-02T05:00:00', end: '2021-12-02T22:00:00', title: 'event 2' },
                    { id: '3', resourceId: 'd', start: '2021-12-06', end: '2021-12-08', title: 'event 3' },
                    { id: '4', resourceId: 'e', start: '2021-12-02T03:00:00', end: '2021-12-02T08:00:00', title: 'event 4' },
                    { id: '5', resourceId: 'f', start: '2021-12-02T00:30:00', end: '2021-12-02T02:30:00', title: 'event 5' }
                ]
            });

            calendar.render();
        });
    });
};`

**Comment by JMackCo** [comment]
This was the piece I was missing:  https://brianlagunas.com/using-npm-packages-in-blazor/

**Comment by contactabdulsami** [comment]
Thanks, I appreciate.

Abdul Sami 
 
  On Mon, 6 Dec 2021 at 8:27 am, ***@***.***> wrote:   


This was the piece I was missing: https://brianlagunas.com/using-npm-packages-in-blazor/

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JMackCo** [comment]
I thought I was on the right track. But, I failed again at getting it to display from a Blazor razor page.

**Comment by JMackCo** [comment]
https://github.com/deandrebaker/fullcalendar-blazor
